compiler/daml-extension/BUILD.bazel

```diff
@@ -106,7 +106,6 @@ genrule(
         tar xzf $$DIR/$(location :node_deps_cache)
         sed -i "s/__VERSION__/{npm}/" package.json
         sed -i 's/"name": "daml"/"name": "daml-bundled"/' package.json
-        $$DIR/$(location //:yarn)
```

Comment:
afaict this was just useless but calling it fails due to some `node-gyp` bs. I did test locally that the extension works as expected.

Reply:
I'm reasonably confident it wasn't useless when it was added, but many things have changed since then so let's see.